config.js

```diff
@@ -0,0 +1,3 @@
+const API_KEY = 'dbb7593505mshdb2b499e2f3acf1p12c4c1jsn93ec8e29d6c6';
+
+export default API_KEY;
```

Comment:
This works well since this is a public API with little risk of losing data but in the future we could look into hiding variables like API keys for better security. We could look into using a module like dotenv and use "environmental variables" instead